### ISSUE TITLE
Regional and static-site analyses ignore remove-trips modification

### ISSUE THREAD

**Issue by ansoncfit** [conveyal/r5#481]
There should be a systematic difference in accessibility for the two scenarios being compared here, but there is only noise:
![image](https://user-images.githubusercontent.com/2173529/49827379-2e907000-fd57-11e8-8d98-4fe5a27189b2.png)

Single-point results show expected differences in isochrones/accessibility.

The route in question is frequency-based, and it is the only route in its feed.

**Comment by ansoncfit** [comment]
In a separate test project that also includes frequency-based routes (https://analysis-staging.conveyal.com/regions/5a2e9df732b98e221fef89eb/projects/5c1086a732b98e5d92c601cc), the regional results are more sensible.  So maybe there is something particular about the feed used.
